_data/projects/apacheshardingsphere-2021.yaml

```diff
@@ -13,3 +13,8 @@ mentors:
     slack: https://opensourceday.slack.com/team/U02D3815D8A
     ask-me: Github, Career advise or life in general :)
     image: tanu-batra.jpg
+   - name: Shraddha Shaligram
```

Comment:
There is an extra space in here that might break the yaml. Please remove it.

Reply:
Updated!